packages/issuer-rest/lib/OID4VCIServer.ts

```diff
@@ -175,7 +175,17 @@ export class OID4VCIServer {
       opts.asClientOpts || this._issuer.asClientOpts ? ({ ...opts.asClientOpts, ...this._issuer.asClientOpts } as ClientMetadata) : undefined
 
     pushedAuthorizationEndpoint(this.router, this.issuer, this.authRequestsData)
-    getMetadataEndpoints(this.router, this.issuer)
+
+    // Create root router for alternative .well-known endpoints if needed
+    const basePath = getBasePath(this.baseUrl)
+    let rootRouter: express.Router | undefined
```

Comment:
I think the dual hosting should have an actual config option to enable/disable (enabled by default)

Reply:
fixed